Blackjack.Tests/RulesShould.cs

```diff
@@ -6,8 +6,11 @@ namespace Blackjack.Tests
     public class RulesShould
     {
         [Theory]
-        [MemberData(nameof(RulesShouldTestData.HandScoreBustTrueData), MemberType = typeof(RulesShouldTestData))]
-        public void ReturnTrue_GivenScoreGreaterThan21(Hand hand, int score)
+        [InlineData(24)]
+        [InlineData(26)]
+        [InlineData(22)]
+        [InlineData(30)]
+        public void ReturnTrue_GivenScoreGreaterThan21(int score)

```

Comment:
Nice `Rules`. As you have two rules `IsBust` and `IsBlackjack`, perhaps the names of the tests could reflect that explicitly.